### DISCUSSION TITLE
I try to define a login function in command.js, then use it ,but the error message is "cy.login is not a function

### DISCUSSION THREAD

**Discussion by guanyingjie** [cypress-io/cypress#16967] [Questions and Help]
this is my command.js
![image](https://user-images.githubusercontent.com/39229309/122318073-1b2a7500-cf51-11eb-8891-e695d7a718c2.png)

this is my test.js
![image](https://user-images.githubusercontent.com/39229309/122318111-28dffa80-cf51-11eb-8557-0addc8db10d0.png)

this is the error message
![image](https://user-images.githubusercontent.com/39229309/122318152-34cbbc80-cf51-11eb-87dc-de4eb37333c6.png)

then I change the import to require like it 
![image](https://user-images.githubusercontent.com/39229309/122318198-44e39c00-cf51-11eb-8190-885f720ccfdf.png)

but still not work

**Comment by bahmutov** [comment]
What’s you support file, does it import or require the commands file? What’s your cypress.json file, does it set the support file to false by any chance?

**Reply by guanyingjie** [reply]
is it the support file?
<img width="458" alt="企业微信截图_e9d123ac-d832-4b91-b4ec-afd797172fee" src="https://user-images.githubusercontent.com/39229309/122319568-59289880-cf53-11eb-82a4-f2241f65f40b.png">
I have import the commands file in to the support file.
And this is my cypress.json
![image](https://user-images.githubusercontent.com/39229309/122321519-90e50f80-cf56-11eb-85cb-33d6a798e1fc.png)
